setup.py

```diff
@@ -37,6 +37,9 @@
 
 INSTALL_REQUIRES = ['numpy >= 1.7', 'pandas >= 0.15.0']
 TESTS_REQUIRE = ['pytest >= 2.7.1']
+if sys.version_info < (3, 0):
```

Comment:
I've added a check for Python < 3 to `setup.py`.  This seems to work.  I'm not sure if this really is an accepted way of creating conditional requirements.